taxi_service/settings.py

```diff
@@ -133,3 +134,9 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+LOGIN_URL = "/login/"
```

Comment:
The `LOGIN_URL` is set to `/login/`. Ensure that this URL matches the login view's URL pattern defined in your `urls.py` to avoid any routing issues.